README.md

```diff
@@ -51,8 +40,28 @@ Image security policies define Portieris' behavior in your cluster. There are tw
 
 ## Configuring image security policies
 
-You can configure custom security policies to control what images can be deployed in your Kubernetes namespaces, and to enforce trust pinning of particular signers. For more information, see the [IBM Cloud docs](https://console.bluemix.net/docs/services/Registry/registry_security_enforce.html#customize_policies).
+You can configure custom security policies to control what images can be deployed in your Kubernetes namespaces, and to enforce trust pinning of particular signers.  
+Portieris supports sourcing trust data from the following registries without additional configuration in the image policy:
+* IBM Cloud Container Registry
+* Quay.io
+* Docker Hub
 
+If users would like to use a different trust server for a repository, you can do so by configuring the policy:  
+*Example*
+```
+apiVersion: securityenforcement.admission.cloud.ibm.com/v1beta1
+kind: ImagePolicy
+metadata:
+  name: allow-custom
+spec:
+   repositories:
+    - name: "registry.bluemix.net/*"
+      policy:
+        trust:
+          enabled: true
+          trustServer: "https://registry.ng.bluemix.net:4443" # Custom trust server for repository
```

Comment:
Could the comment here point out that the trust server is optional?

Reply:
Done